specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+      "post": {
+        "operationId": "CassandraCluster_RequestRepair",
+        "description": "Request that repair begin on this cluster as soon as possible.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraRepair": {
+            "$ref": "./examples/CosmosDBManagedCassandraRepair.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The repair request was successfully received."
+          },
+          "409": {
+            "description": "Repair is disabled for this cluster, so no repair request can be accepted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/instanceView": {
+      "get": {
+        "operationId": "CassandraCluster_GetInstanceView",
+        "description": "Request that repair begin on this cluster as soon as possible.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraClusterInstanceView": {
+            "$ref": "./examples/CosmosDBManagedCassandraClusterInstanceView.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Update will proceed asynchronously."
+          },
+          "200": {
+            "description": "The repair request was successfully received.",
+            "schema": {
+              "$ref": "#/definitions/ClusterInstanceView"
+            }
```

Comment:
ClusterInstanceView is not modelled as a resource. In the URI instanceView should be plural and should be a list call. If is a point GET then it should have a param to provide the name of the instanceView. If InstanceView is always a single value then it uri should look like ....../instanceViews/default